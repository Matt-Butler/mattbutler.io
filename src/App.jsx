import React from 'react';
import { Link, Route } from 'react-router-dom';
import Page1 from './components/Page1';
import Page2 from './components/Page2';
import Home from './components/Home';

const App = () => (
  <div>
    <ul>
      <li>
        <Link to="/home">Home</Link>
      </li>
      <li>
        <Link to="/page1">Page 1</Link>
      </li>
      <li>
        <Link to="/page2">Page 2</Link>
      </li>
    </ul>
    <Route path="/home" component={Home} />
    <Route path="/page1" component={Page1} />
    <Route path="/page2" component={Page2} />
  </div>
);

export default App;
