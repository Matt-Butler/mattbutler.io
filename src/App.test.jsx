import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import App from './App';

it('renders without crashing', () => {
  shallow(<App />);
});

it('renders correctly', () => {
  const wrapper = shallow(<App />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
