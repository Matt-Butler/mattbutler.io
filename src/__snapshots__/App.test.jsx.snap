// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h2>
      mattbutler.io
    </h2>
  </div>
</div>
`;
